gapis/api/vulkan/vulkan.go

```diff
@@ -162,6 +158,7 @@ const (
 	DebugMarker = iota
 	RenderPassMarker
 	DrawGroupMarker
+	otherGroupMarker
```

Comment:
Why is this one private, while the others aren't. And what does 'other' mean? can you give it a better name?

Reply:
Here, `other` means  literally `other`. Any grouping that is not a draw, renderpass or debug. It is only used in one place which says:

```
// Handle an edge case where a group of state setting commands are
// followed by something other than a drawing command.
```

This used to be tagged as a draw group even though it does not draw. Therefore I wanted to separate it so we don't mistakenly tag a non-draw command a draw command.

If you have any other name suggestion, I am happy to accept.

Reply:
And actually all of them should be private. I will change them to private.